GettingStarted/factors.CPP

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include <conio.h>
+using namespace std;
+
+int main()
+{
+    int n, i;
+
+    cout << "Enter a positive integer: ";
+    cin >> n;
+
+    cout << "Factors of " << n << " are: " << endl;  
+    for(i = 1; i <= n; ++i)
```

Comment:
```suggestion
    for(i = 1; i <= n/2; ++i)
```
Factors of a number n can be upto n/2 except the number itself. Also add `cout<<n<<endl;` outside the loop.